doc/source/api/executors/awslambda.rst

```diff
@@ -7,23 +7,193 @@
+     - Duration in seconds before the lambda times out
+   * - memory_size
+     - Yes
+     - 512
+     - Amount in MB of memory to allocate to the lambda
+   * - poll_freq
+     - No
+     - 5
+     - Time interval between successive polls to the lambda function
+   * - cache_dir
+     - No
+     - ~/.cache/covalent
+     - Path on the local file system to a cache
+   * - cleanup
+     - No
+     - True
+     - Flag represents whether or not to cleanup temporary files generated during execution
+
+The following snippet shows how users may modify their Covalent `configuration <https://covalent.readthedocs.io/en/latest/how_to/config/customization.html>`_ to provide
+the necessary input arguments to the executor:
+
+.. code:: bash
+
+    [executors.awslambda]
+    credentials = "/home/<user>/.aws/credentials"
+    profile = "default"
+    region = "us-east-1"
+    lambda_role_name = "CovalentLambdaExecutionRole"
+    s3_bucket_name = "covalent-lambda-job-resources"
+    cache_dir = "/home/<user>/.cache/covalent"
+    poll_freq = 5
+    timeout = 60
+    memory_size = 512
+    cleanup = true
+
+Within a workflow, users can use this executor with the default values configured in the configuration file as follows:
+
+.. code:: python
+
+    import covalent as ct
+
+    @ct.electron(executor="awslambda")
+    def task(x, y):
+        return x + y
+
+
+Alternatively, users can customize this executor entirely by providing their own values to its constructor as follows:
+
+.. code:: python
+
+    import covalent as ct
+    from covalent.executor import AWSLambdaExecutor
+
+    lambda_executor = AWSLambdaExecutor(credentials="my_custom_credentials",
+                                profile="custom_profile",
+                                region="us-east-1",
+                                lambda_role_name="my_lambda_role_name",
+                                s3_bucket_name="my_s3_bucket",
+                                cache_dir="/home/<user>/covalent/cache",
+                                poll_freq=5,
+                                timeout=30,
+                                memory_size=512,
+                                cleanup=True)
+
+    @ct.electron(executor=lambda_executor)
+    def task(x, y):
+        return x + y
+
+.. _required_aws_resources:
+
+4. Required AWS Resources
+###########################
+
+In order for the executor to work end-to-end, the following resources need to be configured
+either with `Terraform <https://www.terraform.io/>`_ or manually provisioned on the `AWS Dashboard <https://aws.amazon.com/>`_:
+
+.. list-table:: Title
+   :widths: 25 25 50
+   :header-rows: 1
+
+   * - Resource
+     - Config Name
+     - Description
+   * - IAM Role
+     - lambda_role_name
+     - The IAM role this lambda will assume during execution of your tasks
+   * - S3 Bucket
+     - s3_bucket_name
+     - Name of an AWS S3 bucket that the executor can use to store temporary files
+
+4a. Manually provisioning resources
```

Comment:
@Emmanuel289 I think just due to aesthetic reasons let's just remove the `4a. Manually provisioning resources` heading (also such that it is consistent with the other executor's updated RTD)

Reply:
> @Emmanuel289 I think just due to aesthetic reasons let's just remove the `4a. Manually provisioning resources` heading (also such that it is consistent with the other executor's updated RTD)

I have effected the requested changes. Thanks!